### PR TITLE
feat: use build env file to load secret environs dynamically

### PR DESCRIPTION
**Pull request by floralatin**

### CODE REVIEW

File: normalizer/docker/parser.py [hunk 1]
```diff
@@ -44,16 +46,27 @@ def __init__(
...
+            if RUN_VAR_RE.match(strip_line):
+                replace_line = line.replace('RUN', f'RUN {build_env_str}')
+                self._parser.content = self._parser.content.replace(line, replace_line)
+
+    def insert_build_env_path(self, build_env_path: pathlib.Path):
+        build_env_str = ''
+        build_env_path_str = str(build_env_path)
+
+        build_env_str += dedent(
+            f' --mount=type=secret,id={build_env_path.stem},dst={build_env_path} . {build_env_path_str} '
```
Comment (nomagick) [inline normalizer/docker/parser.py:69]: The id and dst here are the same is this valid?
I think we can simply presume the secret name be `hubble_build_env` and load from `/run/secrets/hubble_build_env`

**Review by nomagick — CHANGES_REQUESTED** [review]

File: normalizer/docker/parser.py (hunk 1 above)
Comment (floralatin) [reply normalizer/docker/parser.py:69]: > The id and dst here are the same is this valid?

mmm 'build_env_path.stem' means: `filename`;
```
build_env_path =  'src/build_env.sh'  
build_env_path.stem is `build_env`
```

Comment (nomagick) [reply normalizer/docker/parser.py:69]: I see. But why we need this anyway, we could simply load from `/run/secrets/{id}` right?

Comment (floralatin) [reply normalizer/docker/parser.py:69]: > I see. But why we need this anyway, we could simply load from `/run/secrets/{id}` right?

I will convert `--mount=type=secret,id={build_env_path.stem},dst={build_env_path} . {build_env_path_str} ` to`--mount=type=secret,id={build_env_file}. /run/secrets/{build_env_file}`

**Pull request merged**